requirements.txt

```diff
@@ -283,8 +283,8 @@ reportlab==3.4.0 \
 requests==2.12.4 \
     --hash=sha256:000748df49e087784441b2621c50fb81046c5c8e80e0d91674ffad65b9e13844 \
     --hash=sha256:ed98431a0631e309bb4b63c81d561c1654822cb103de1ac7b47e45c26be7ae34
-SQLAlchemy==1.0.12 \
-    --hash=sha256:6679e20eae780b67ba136a4a76f83bb264debaac2542beefe02069d0206518d1
+SQLAlchemy==1.1.14 \
```

Comment:
Any reason why we updated SQLAlchemy as part of this PR?

Reply:
My bad, tried a change elsewhere and hashin added it here, will remove